test/e2e/common/node/container_probe.go

```diff
@@ -235,6 +235,10 @@ var _ = SIGDescribe("Probing container", func() {
 		Description: A Pod is created with readiness probe with a Exec action on the Pod. If the readiness probe call does not return within the timeout specified, readiness probe MUST not be Ready.
 	*/
 	ginkgo.It("should not be ready with an exec readiness probe timeout [MinimumKubeletVersion:1.20] [NodeConformance]", func() {
+		// The ExecProbeTimeout feature gate exists to allow backwards-compatibility with pre-1.20 cluster behaviors, where readiness probe timeouts were ignored
+		// If ExecProbeTimeout feature gate is disabled, timeout enforcement for exec readiness probe is disabled, so we should skip this test
+		e2eskipper.SkipUnlessExecProbeTimeoutEnabled()
```

Comment:
I seem to recall that skipping based on feature flags is not allowed for conformance tests.

cc @chewong @dims @derekwaynecarr 

if I'm wrong lgtm

Reply:
skipping is not allowed for `[Conformance]` but this is not a conformance test (which would use `framework.ConformanceIt`)

Reply:
Asked on the Conformance slack channel and it seems NodeConformance is targeted at CRI conformance, not Kubernetes conformance, so this should be okay. https://kubernetes.slack.com/archives/C78F00H99/p1623803503028500